src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockType.java

```diff
@@ -53,34 +53,36 @@ public enum BlockType {
 	private final BiFunction<Block, QuiltBlockSettings, Block> creator;
 	private final float strength;
 	private final String[] variants;
+	private final SafeRenderLayer renderLayer;
 
 	BlockType(BiFunction<Block, QuiltBlockSettings, Block> creator, float strength, String[] variants, SafeRenderLayer renderLayer) {
 		this.creator = creator;
 		this.strength = strength;
 		this.variants = variants;
+		this.renderLayer = renderLayer;
 	}
 
 
 	public String[] variants() {
 		return variants;
 	}
 
+	public SafeRenderLayer renderLayer() {
+		return renderLayer;
+	}
+
 	@Override
 	public String toString() {
 		return name().toLowerCase(Locale.ROOT);
 	}
 
 	public TypedGroupedBlock register(BlockGroup group, BlockGroup.GroupedBlock groupedBlock, BlockCreationCallback callback) {
-		return register(group, groupedBlock, callback, ArchitectureExtensions.MOD_CONTAINER.metadata().id()); // If no id is specified, use our own id
```

Comment:
I want to retain the modid of the requester for as long as possible for error reporting

Reply:
👍